### PR TITLE
Bash variable naming

### PR DESCRIPTION
**Pull request by sunsingerus**
Pull Request for this [issue](https://github.com/timescale/tsbs/issues/9)
Some scripts refactoring and unification for more convenient usage

### CODE REVIEW

File: scripts/load_cassandra.sh [hunk 1]
```diff
@@ -1,20 +1,32 @@
...
+    echo "tsbs_load_cassandra not available. It is not specified explicitly and not found in \$PATH"
+    exit 1
+fi
+
+# Load parameters - common
 DATA_FILE_NAME=${DATA_FILE_NAME:-cassandra-data.gz}
+DATABASE_PORT=${DATABASE_PORT:-9042}
+
+# Load parameters - personal
 PROGRESS_INTERVAL=${PROGRESS_INTERVAL:-20s}
```
Comment (RobAtticus) [inline scripts/load_cassandra.sh]: Actually we can add PROGRESS_INTERVAL to the common file, it should be `10s` for all of them.

**Comment by RobAtticus** [comment]
Also can you squash your commits into one?

File: scripts/load_cassandra.sh (hunk 1 above)
Comment (sunsingerus) [reply scripts/load_cassandra.sh]: Sounds reasonable, I'll move PROGRESS_INTERVAL to common file and change to 10 sec

**Comment by sunsingerus** [comment]
> Also can you squash your commits into one?

yes, let me move PROGRESS_INTERVAL and squash all commits

**Comment by RobAtticus** [comment]
Awesome, thanks a bunch for this!

**Comment by RobAtticus** [comment]
Hey @sunsingerus , hate to bother, but I didn't realize I forgot to enable our contributor licensing on this repo. Is it possible for you to sign it belatedly?:
https://cla-assistant.io/timescale/tsbs

**Pull request merged**